ios/Plugin/Plugin.swift

```diff
@@ -554,4 +554,27 @@ public class RadarPlugin: CAPPlugin, RadarDelegate {
         }
     }
 
+    @objc func setLogLevel(_ call: CAPPluginCall) {
+        DispatchQueue.main.async {
+            guard let levelStr = call.getString("level") else {
```

Comment:
Are there enum cases or constants, or do we just use bare strings? If the latter, we may want to do case-insensitive comparisons.

Reply:
The strings are coming in from the web-app, to match enums in android and iOS SDKs. I just updated the comparisons to use case insensitive comparisons, as suggested.

Reply:
👍